src/internal/context/split-panel-context.ts

```diff
@@ -38,12 +39,16 @@ export interface SplitPanelContextProps extends SplitPanelContextBaseProps {
   animationDisabled?: boolean;
 }
 
-const SplitPanelContext = createContext<SplitPanelContextProps | null>(null);
+const AppLayoutWidgetSplitPanelContext =
+  awsuiPluginsInternal.sharedReactContexts.createContext<SplitPanelContextProps | null>(
```

Comment:
We use this utility for similar use-cases in app layout widget already: https://github.com/cloudscape-design/components/blob/main/src/app-layout/visual-refresh-toolbar/contexts.ts

Now it is time to onboard split panel context as well (also to support app layout widget changes, actually)